sdk/core/loader/boot.cc

```diff
@@ -831,6 +865,7 @@ namespace
 			        false>(config.trustedStack);
 			threadTStack->mepcc = pcc;
 			threadTStack->cgp   = cgp;
+			threadTStack->cra   = initialCRA;
```

Comment:
Please could you update `fail-simulator-on-error.h` to account for this? Should be able to delete some code :-)